ml/src/main/scala/ai/h2o/sparkling/ml/models/H2OTargetEncoderModel.scala

```diff
@@ -69,12 +69,11 @@ class H2OTargetEncoderModel(override val uid: String, targetEncoderModel: H2OMod
     val params = Map(
       "model" -> targetEncoderModel.modelId,
       "frame" -> input.frameId,
-      "data_leakage_handling" -> getHoldoutStrategy(),
       "noise" -> getNoise(),
```

Comment:
I see, `noise` is provided only at `transform` level, which is where it is needed technically.
Although providing it at training level allows to define a default noise value that will then be used for all subsequent calls to `transform`: this may be more convenient for some scenarios.